enhancements/20190919-OVN-Kubernetes.md

```diff
@@ -48,39 +48,50 @@ ovn-northd, which converts logical objects in to flows.
 The data store is a replicated database, with a traditional master-backup
 architecture.
 
-The node-level process (ovn-southd) is responsible for programming the
+The node-level process (ovn-controller) is responsible for programming the
 flows from ovn-northd in to the vswitch on the local node. It also handles
 some bookkeeping.
 
 #### Controller
 There will be a controller daemonset on every master node. They will
 perform leader election directly via the Kubernetes API. The ovn-kubernetes
 process will, when it achieves leadership, will be responsible for
-starting the ovn-north process.
+starting the ovn-northd process.
 
 #### Data store
 We will write a small controller that, based on the result of the leader
 election process, configures the databases (ovn-ndbd and ovn-sbdb) as
 master or backup. The master will be distributed via an endpoint. If a
 node is backup, it will watch for endpoint changes and update replication
-configuration as fit.
+configuration accordingly.
 
 #### Node-level
 The node-level components are configured as a daemonset. They run on all
 nodes, including the control-plane. They watch for endpoint changes,
 watch the sbdb for changes, and configure the nodes.
 
 ### Test Plan
-OVN-Kubernetes must pass all standard QE tests, as well as the full e2e
-suite. It does not need any special testing, as the GA release does not
-enable any new features.
+In addition to the existing CI and QE test suites, we will develop both manual and
```

Comment:
aside: we need e2e-aws-upgrade-ovn-kubernetes (*after* we implement the HA stuff)